servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -808,6 +816,7 @@ private FileBlockInfo generateFileBlockInfo(InodeFile file, BlockInfo blockInfo)
         }
       }
     }
+    MasterContext.getMasterSource().incFileBlockInfosGot(1);
```

Comment:
Instead of doing this for each FileBlockInfo, should this be incremented in batch where the lists of these objects are returned?